### DISCUSSION TITLE
Inviting a User Through the Supabase Dashboard

### DISCUSSION THREAD

**Discussion by kennethslee613** [supabase/supabase#16423] [Questions] [answered]
Hi everyone!

I'm trying to implement a way to invite a user to sign up by emailing them an invite link. More specifically, I'm trying to do it through the admin panel in the Authentication tab, but I'm getting an error when clicking on Add user -> Send invitiation:

```
Failed to invite user: failed to make invite request: null value in column "username" of relation "profiles" violates not-null constraint
```

My user database requires a username to exist for every user, but I can't make a username without creating a row for the user, which means the user id and user email needs to exist. Is there a way to use the Send invitation feature and also pass in data like the `username`? Or how can I get the Send invitation feature to work?

Thank you!

**Comment by GaryAustin1** [comment, marked as answer]
You would need to modify your profiles table to allow null and then have the user enter that data later using updateUser().

**Reply by kennethslee613** [reply]
Got it, thank you!
